src/WebJobs.Extensions.DocumentDB/Trigger/CosmosDBTriggerAttribute.cs

```diff
@@ -87,5 +82,47 @@ public CosmosDBTriggerAttribute(string databaseName, string collectionName)
         /// collection.
         /// </summary>
         public int LeasesCollectionThroughput { get; set; }
+
+        /// <summary>
+        /// Optional.
+        /// Defines a prefix to be used within a Leases collection for this Trigger. Useful when sharing the same Lease collection among multiple Triggers
+        /// </summary>
+        public string LeaseCollectionPrefix { get; set; }
+
+        /// <summary>
+        /// Optional.
+        /// Customizes the amount of milliseconds between lease checkpoints
+        /// </summary>
+        public int CheckpointFrequency { get; set; }
```

Comment:
Request that you also expose they "checkpoint every x documents" from ChangeFeedHostOptions.

Reply:
@nkinnan Thanks! It's on the `MaxItemPerInvocation` attribute.